test/opt/aggressive_dead_code_elim_test.cpp

```diff
@@ -5161,7 +5160,6 @@ OpMemoryModel Logical GLSL450
 OpEntryPoint Fragment %main "main" %output
 OpExecutionMode %main OriginUpperLeft
 OpDecorate %1 Restrict
-OpDecorate %1 Aliased
```

Comment:
I deleted _OpDecorate %1 Aliased_ to stop it from failing. Would you double check if I need to replace with something else or the test is still valuable?